src/rt/sections_osx.d

```diff
@@ -23,6 +23,13 @@ import core.sys.osx.mach.getsect;
 import rt.deh, rt.minfo;
 import rt.util.container.array;
 
+version (D_LP64)
+    import rt.sections_osx64;
+else
+    import rt.sections_osx32;
+
+mixin SectionsOSX;
```

Comment:
I probably would have opted for creating a third/common file. The mixin approach has two problems: The mixin could change meaning depending on where it's used, and while editing the contents of the mixin, intellisense systems will not know where to look for the symbols. Newcomers looking at sections_osxXX may also be confused as well. I'm not sure how prevalent this type of thing is in the druntime code, but if it can be avoided, I think it should.